src/main/kotlin/com/linchpino/core/entity/MentorInterviewType.kt

```diff
@@ -1,7 +1,15 @@
 package com.linchpino.core.entity
 
-import jakarta.persistence.*
-import lombok.*
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import lombok.AllArgsConstructor
+import lombok.Builder
+import lombok.Getter
+import lombok.NoArgsConstructor
+import lombok.Setter
 
 @Entity
 @Table(name = "MENTOR_INTERVIEW_TYPE")
```

Comment:
I think this entity should be removed. I have created a table to present a "many to many" relation between mentor (account) and @SMSaeedi  created it as an entity